### PR TITLE
Multiline support

### PR DESCRIPTION
**Pull request by swieka**
Commits 1-3 (1da892f - 8b86e24):
- adding docker support with express server for quick run (without any development environment)
- adding multi-line text example page

Commit 4-5 (36af266 -0fe4237) adding multi-line centered text support.

The 5th commit may be used as a base for signature extension allowing to specify alignment.

I use multi-line text in 3d-force-graph displaying 2 line text nodes.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -53,14 +53,19 @@ export default class extends three.Sprite {
     const font = `${this.fontWeight} ${this.fontSize}px ${this.fontFace}`;
 
     ctx.font = font;
-    const textWidth = ctx.measureText(this.text).width;
+	
+	var lines = this._text.split('\n');
```
Comment (vasturiano) [inline src/index.js]: Please use ES6 syntax (`const`, lambda fns (`=>`)) to remain idiomatic with the existing code.

File: src/index.js [hunk 2]
```diff
@@ -53,14 +53,19 @@ export default class extends three.Sprite {
...
     canvas.width = textWidth;
-    canvas.height = this.fontSize;
+    canvas.height = this.fontSize * lines.length;
 
     ctx.font = font;
     ctx.fillStyle = this.color;
     ctx.textBaseline = 'bottom';
-    ctx.fillText(this.text, 0, canvas.height);
+	
+	for (var i = 0; i<lines.length; i++)
```
Comment (vasturiano) [inline src/index.js]: Please use `lines.forEach(line => ...)` for clearer syntax.

**Comment by vasturiano** [comment]
Thanks for the PR @swieka. Could you please remove all the docker / express scaffolding? It's not appropriate for this module.

**Comment by vasturiano** [comment]
I'm wondering if it wouldn't be easier to use a library like [canvas-txt](https://github.com/geongeorge/Canvas-Txt) to handle line breaks. And as a bonus also provides text alignment etc.

**Comment by swieka** [comment]
> Thanks for the PR @swieka. Could you please remove all the docker / express scaffolding? It's not appropriate for this module.

Removed

**Comment by swieka** [comment]
> I'm wondering if it wouldn't be easier to use a library like [canvas-txt](https://github.com/geongeorge/Canvas-Txt) to handle line breaks. And as a bonus also provides text alignment etc.

I think it is an implementation detail hidden from external usage. I propose to merge current solution and consider replacement as a next improvement.

**Comment by vasturiano** [comment]
> > I'm wondering if it wouldn't be easier to use a library like [canvas-txt](https://github.com/geongeorge/Canvas-Txt) to handle line breaks. And as a bonus also provides text alignment etc.
> 
> I think it is an implementation detail hidden from external usage. I propose to merge current solution and consider replacement as a next improvement.

Yep, agreed. It works fine with the current implementation and it neatly center aligns the text, so no need to add a dependency.

I've just fixed a small issue with the sprite scale so that text size remains constant across multiple lines. New version published: `1.2.0`.

Thanks for the PR. 👍

**Pull request merged**